earth_enterprise/src/fusion/autoingest/AssetFactory.h

```diff
@@ -49,210 +55,181 @@ namespace AssetFactory
       return nullptr;
   }
 
-  template<class AssetType>
```

Comment:
Most of the changes in this file are to use new template parameters that make the code clearer and make it easier to use the right template parameters.